corehq/apps/fixtures/dispatcher.py

```diff
@@ -19,4 +19,5 @@ class FixtureInterfaceDispatcher(ProjectReportDispatcher):
 
     @method_decorator(require_can_edit_fixtures)
     def dispatch(self, request, *args, **kwargs):
+        request.use_requirejs = True
```

Comment:
There's a smell here. Python should not need to know about requirejs. Is the long-term vision to remove these once requirejs is used in the majority of places?

This is not the first time in this PR that I noticed this, but I couldn't articulate the problem before (not even sure I've done a good job here).

Reply:
Consider a custom template tag for `requirejs_entry_point` that would set an attribute like `requirejs_entry` on the request - and, by convention, would be set at the top of the template after loading template tag libraries - and then in base.html, could use `{{ request.requirejs_entry }}` to get the file name and test for `{% if request.requirejs_entry %}` to determine whether or not to include script tags.

(and maybe rename `requirejs_entry`)

Reply:
https://github.com/dimagi/commcare-hq/pull/17824/commits/c34703accfb1f66547ada7faa10bb7fc52e61a44